README.md

```diff
@@ -9,6 +9,7 @@ Following files are present in this directory. You may contribute to any of thes
 Potential Topics--
 
 - Tech stacks
+    1. Nuxt3. 
```

Comment:
Please provide context by making it a link to a new file and explaining what it is and why it may be used (you can use another main link for that in the Nuxt3.md page or something)